benchmarks/benchmark_run_tardis.py

```diff
@@ -2,19 +2,39 @@
 import os
 from tardis.io.configuration.config_reader import Configuration
 from tardis import run_tardis
+from astropy import units as u
 
 class Benchmarkruntardis:
     """Class to benchmark the run_tardis function.
     """
     timeout = 200
-    
-    def setup(self):
+    params = ['branch85_w7', 'uniform','power_law','exponential']
+    param_names = ['Density']
+
+    def setup(self, d):
```

Comment:
`d` is not a good variable name, please use something more descriptive

Reply:
I have changed it.